Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM        sdurrheimer/alpine-golang-make-onbuild
+FROM        prom/busybox:glibc
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
-EXPOSE      9100
+COPY node_exporter  /bin/node_exporter
+
+EXPOSE     9100
+ENTRYPOINT [ "/bin/node_exporter" ]
```

Comment:
Ok I got it.

One more thing to take into account, by removing the entrypoint, we might break a lot a prometheus docker's users if they update their images but don't change their custom CMD to include the binary path in front of it.